config.yaml

```diff
@@ -9,6 +9,7 @@ default:
  arch_and_endianness: 'ppc64le'
  release_notes_repo_url: "https://github.com/open-power-host-os/open-power-host-os.github.io.git"
  release_notes_repo_branch: "master"
+ mock_binary: "/usr/bin/mock"
```

Comment:
Is this a requirement? Is there a concrete use case (have you ever had the need of use an alternate Mock binary or found a mock rpm package which installs it in a different path) ?